Library/Homebrew/utils/analytics.rb

```diff
@@ -1,67 +1,73 @@
-def analytics_label
-  @analytics_anonymous_prefix_and_os ||= begin
-    os = OS_VERSION
-    prefix = ", non-/usr/local" if HOMEBREW_PREFIX.to_s != "/usr/local"
-    ci = ", CI" if ENV["CI"]
-    "#{os}#{prefix}#{ci}"
-  end
-end
+module Utils
+  module Analytics
+    class << self
```

Comment:
One motivation might be that RuboCop by default complains about `extend self`, though that is quite a widely used pattern in Homebrew.